Mu2eG4/geom/geom_common_current_ExtendedDiffusionPump.txt

```diff
@@ -0,0 +1,15 @@
+// A geometry that uses all the latest Offline code
+// and features & Extended Diffusion pump in the RHR for a radiation dose study.
+
+#include "Offline/Mu2eG4/geom/geom_2021_PhaseI_v03.txt"
```

Comment:
Please change this to include geom_common_current, so that it will stay in sync as other changes are made to the geometry.

Reply:
done.